openapi/paths/tracking@taxes.yaml

```
@@ -0,0 +1,36 @@
+get:
+  x-products:
+    - Users
+  tags:
+    - Tracking
+  summary: Retrieve tracking tax logs
+  operationId: GetTaxLogCollection
+  x-sdk-operation-name: getAllTaxLogs
```

nitpick: I think we may be missing a description here
```suggestion
  x-sdk-operation-name: getAllTaxLogs
  description: Retrieves tracking tax logs.
```